src/redux/actions/authActions.ts

```diff
@@ -0,0 +1,63 @@
+import { ActionTypes } from "./actionTypes";
+import { IPayloadAction, createPayloadAction, createAction } from "./actionCreators";
+import { IAuth } from "../../models/applicationState";
+import { Dispatch, Action } from "redux";
+import { IpcRendererProxy } from "../../common/ipcRendererProxy";
+
+/**
+ * Actions which manage users auth
+ * @member signIn - Allows to sign in to the application
+ * @member signOut - Allows to sign out from the application
+ */
+export default interface IAuthActions {
```

Comment:
How this interface is used ?

Reply:
Right not it's not used but I believe that it will be unnecessary in the future, on other tasks (login to the application etc.)